index.html

```diff
@@ -71,15 +76,15 @@ <h3>give me a random one</h3>
         </div>
     </div>
     <div class="tokio">
```

Comment:
Please remove similar screens such as Tokyo, Miami, etc. For further work we need only one screen, which will display content conditionally: display screen with search by default and change it with styles and JS to screen with weather info.